parallel-install/pkg/components/components.go

```diff
@@ -65,15 +67,8 @@ func (p *ComponentsProvider) GetComponents() ([]KymaComponent, error) {
 
 	var components []KymaComponent
 	for _, component := range installationCR.Spec.Components {
-		component := KymaComponent{
-			Name:            component.Name,
-			Namespace:       component.Namespace,
-			OverridesGetter: p.overridesProvider.OverridesGetterFunctionFor(component.Name),
-			ChartDir:        path.Join(p.resourcesPath, component.Name),
-			HelmClient:      helmClient,
-			Log:             p.log,
-		}
-		components = append(components, component)
+		cmp := NewComponent(component.Name, component.Namespace, p.profile, path.Join(p.resourcesPath, component.Name), p.overridesProvider.OverridesGetterFunctionFor(component.Name), helmClient, p.log)
```

Comment:
Same as above.. maybe worth to switch to a param-struct instead passing so many parameter to a function call. this is pretty hard to read.